.travis.yml

```diff
@@ -6,8 +6,9 @@ python:
 install: pip install -r requirements.txt
 
 script:
-- nosetests --with-coverage --cover-package=app
+- nosetests tests/test_app_tests.py --with-coverage
```

Comment:
The test file should be named as 'test_app.py' and not 'test_app_tests.py' . Ideally, the name of a test file should begin with 'test' followed by an underscore and the the name of the file for which the tests are being written. So, you should have something like test-views.py and not test_views_tests.py

Reply:
Okay. I will look into that.